Makefile.rpmbuilder

```diff
@@ -120,7 +120,12 @@ dist-prep:
 dist-build-dep: $(CHROOT_DIR)$(DIST_SRC)/$(PACKAGE)
 
 ifdef INCREMENT_DEVEL_VERSIONS
-dist-package: devel_ver_path = $(ORIG_SRC)/$(OUTPUT_DIR)/$(notdir $(PACKAGE)).devel
+DEVEL_VER_FILE = $(ORIG_SRC)/$(OUTPUT_DIR)/$(notdir $(COMPONENT)).devel
+# Ensure folder present for touch
+$(shell mkdir -p $(ORIG_SRC)/$(OUTPUT_DIR) && touch $(DEVEL_VER_FILE))
```

Comment:
This will execute even if no build it being done. Generally using `$(shell ...)` to make some changes is almost always a bad idea - it should be part of some make target.